webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -166,6 +166,10 @@ const ExperimentTrials: React.FC<Props> = ({ experiment, pageRef }: Props) => {
       );
     };
 
+    const logRetentionDaysRenderer = (_: string, record: TrialItem): React.ReactNode => {
```

Comment:
showing `-1` is confusing, also good to add unit like `days` etc

<img width="192" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/6551c933-1993-46df-a086-738af6656d8e">

Reply:
Done!